eng/Tools/DiagConfig/DiagConfig.csproj

```diff
@@ -14,8 +14,13 @@
     <PackageReference Include="System.CommandLine" />
     <PackageReference Include="System.CommandLine.NamingConventionBinder" />
     <PackageReference Include="Microsoft.CodeAnalysis.Common" />
-    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" />
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp" />
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" />
+    <PackageReference Include="Microsoft.CodeAnalysis.ExternalAccess.AspNetCore" />
+    <PackageReference Include="Microsoft.CodeAnalysis.Features" />
     <PackageReference Include="YamlDotNet" />
   </ItemGroup>
```

Comment:
Why are these new references needed? DiagConfig has been building fine without these...

Reply:
The inbox ASP.NET Core analyzers now depend on this.

Reply:
But Diagconfig doesn't need to reference any analyzers itself in order to build DiagConfig, right? DiagConfig explicitly loads analyzers at runtime.